HW_5/task_1.py

```diff
@@ -0,0 +1,12 @@
+"""
+Напишите программу, удаляющую из текста все слова, в которых присутствуют буквы «а», «б» и «в».
+
+Ввод: значение типа <str>
+Вывод: значение типа <str>
+"""
+"""oldstr = str(input("ВВедите слова: "))
+newstr = str.replace('а') 
+print(newstr)"""
+
+str = str(input("ВВедите слова: "))
```

Comment:
Функция `input` уже возвращает строку.
```suggestion
str = input("Введите слова: ")
```

Reply:
Я уже неоднократно повторял, что не стоит называть переменные заимствованными словами. У вас переменная `str` содержит объект типа `str`.